Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1443,9 +1441,6 @@ def __init__(self, iterable=[]):
 	def array(self):
 		return self._a
 
-	def isFloat(self):
```

Comment:
we may as well keep it since it wasn't prefixed with an underscore and tools in the wild may be accessing that method for whatever reason. bonus point if we add a deprecation warning as well

Reply:
The type was rather internal that's why I have no problem removing it. Any code using it needs to be updated probably. But sure, adding back with warning.

Is ugly that we'll keep forever.

Reply:
actually, it's fine for me if you want to remove it. I doubt anybody uses that